src/ucp/wireup/select.c

```diff
@@ -2006,6 +2000,7 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
             if (added_lanes == added_lanes_prev) {
                 break;
             }
+            max_lanes -= bw_info.max_lanes;
```

Comment:
I think we need to remove the actual number of lanes we have
for example, we can avoid updating max_lanes here, and line 1938 would be:
`bw_info.max_lanes = ucs_min(max_lanes - added_lanes, lanes_batch);`
or, we can add new local variable "num_lanes" that would save the value returned from ucp_wireup_add_bw_lanes, and here we would do:
`max_lanes -= num_lanes`

Reply:
I'm not sure if I understand. The loop condition won't change if we don't update `max_lanes`. Also, decreasing `max_lanes` by `added_lanes` (instead of `bw_info.max_lanes`) will result in additional and unnecessary iteration of the `while` loop because `added_lanes` can be smaller than `bw_info.max_lanes`.